lib/phoenix/digester.ex

```diff
@@ -119,7 +119,7 @@ defmodule Phoenix.Digester do
 
   defp compiled_file?(file_path) do
     Regex.match?(@digested_file_regex, Path.basename(file_path)) ||
-      Path.extname(file_path) == ".gz" ||
+      Path.extname(file_path) in [".gz", ".br"] ||
```

Comment:
isn't there any way to make this configurable? I mean, no problem on `[".gz", ".br"]` being the default config value, but I can see how this could be an useful compile time config

Reply:
`Plug.Static` only supports `.gz` and `.br` right now. Do you have other use cases in mind?

Reply:
Oh, never mind, I though this was just a blacklist of files to not digest, now I read the actual function name and I think it does not make sense to allow people to configure it. Sorry for bothering